test/__snapshots__/serialize.test.ts.snap

```diff
@@ -12380,6 +12380,1890 @@ function __f0() {
+
+return function /*preWarm*/(prewarmContext) {
+                prewarmContext.getOrInit(function_prewarm_1.PrewarmClients.DYNAMO);
+            };
+
+    }
+  }).apply(undefined, undefined).apply(this, arguments);
+}
+
+function __f5(__0, __1) {
+  return (function() {
+    with({ function_prewarm_1: __function_prewarm_1 }) {
+
+return async (args, preWarmContext) => {
+                    const dynamo = preWarmContext.getOrInit(function_prewarm_1.PrewarmClients.DYNAMO);
+                    const [input] = args;
+                    const { Table: table, AttributesToGet, ...rest } = input;
+                    const payload = {
+                        ...rest,
+                        AttributesToGet: AttributesToGet,
+                        TableName: table.tableName,
+                    };
+                    return dynamo.getItem(payload).promise();
+                };
+
+    }
+  }).apply(undefined, undefined).apply(this, arguments);
+}
+
+function __f6(__0) {
+  return (function() {
+    with({ function_prewarm_1: __function_prewarm_1 }) {
+
+return function /*preWarm*/(prewarmContext) {
+                prewarmContext.getOrInit(function_prewarm_1.PrewarmClients.DYNAMO);
+            };
+
+    }
+  }).apply(undefined, undefined).apply(this, arguments);
+}
+
+function __f7(__0, __1) {
+  return (function() {
+    with({ function_prewarm_1: __function_prewarm_1 }) {
+
+return async (args, preWarmContext) => {
+                    const dynamo = preWarmContext.getOrInit(function_prewarm_1.PrewarmClients.DYNAMO);
+                    const [input] = args;
+                    const { Table: table, ...rest } = input;
+                    return dynamo
+                        .updateItem({
+                        ...rest,
+                        TableName: table.tableName,
+                    })
+                        .promise();
+                };
+
+    }
+  }).apply(undefined, undefined).apply(this, arguments);
+}
+
+function __f8(__0) {
+  return (function() {
+    with({ function_prewarm_1: __function_prewarm_1 }) {
+
+return function /*preWarm*/(prewarmContext) {
+                prewarmContext.getOrInit(function_prewarm_1.PrewarmClients.DYNAMO);
+            };
+
+    }
+  }).apply(undefined, undefined).apply(this, arguments);
+}
+
+function __f9(__0, __1) {
+  return (function() {
+    with({ function_prewarm_1: __function_prewarm_1 }) {
+
+return async (args, preWarmContext) => {
+                    const dynamo = preWarmContext.getOrInit(function_prewarm_1.PrewarmClients.DYNAMO);
+                    const [input] = args;
+                    const { Table: table, Item, ...rest } = input;
+                    return dynamo
+                        .putItem({
+                        ...rest,
+                        Item: Item,
+                        TableName: table.tableName,
+                    })
+                        .promise();
+                };
+
+    }
```

Comment:
Does it pull in all of the table's native methods even if they aren't used?

Reply:
Yes, originally I thought this was due to the namespace, but I changed to a constant and it still happened. Not sure why it cannot tree shake here.